glfw/src/vulkan.zig

```diff
@@ -109,7 +109,7 @@ pub const VKProc = fn () callconv(.C) void;
 /// @pointer_lifetime The returned function pointer is valid until the library is terminated.
 ///
 /// @thread_safety This function may be called from any thread.
-pub fn getInstanceProcAddress(vk_instance: ?*opaque {}, proc_name: [*:0]const u8) callconv(.C) ?VKProc {
+pub fn getInstanceProcAddress(vk_instance: ?*anyopaque, proc_name: [*:0]const u8) callconv(.C) ?VKProc {
```

Comment:
Do you mind updating the other two instances of `*opaque {}` below in the `getPhysicalDevicePresentationSupport` method?

Reply:
Good spot; just pushed this because the only thing I was using was this first one, slipped my mind to consider the other two instances.